app/assets/javascripts/preferences/components/menu-item.tsx

```diff
@@ -0,0 +1,27 @@
+import { Icon, IconType } from '@/components/Icon';
```

Comment:
For components, let's name the file in camelcase (MenuItem in this case), to keep it consistent with the rest of the project